jaxrs-spec/src/main/asciidoc/chapters/providers/_providers.adoc

```diff
@@ -20,6 +20,15 @@ and MAY be annotated with `@Provider` for automatic discovery purposes;
 the integration of pre-packaged providers into the JAX-RS runtime is
 implementation dependent.
 
+[[extensions]]
+Providers MAY be annotated with `@Extension` to declare it as an
```

Comment:
This sounds like `Extension` is an annotation, but actually it is some kind of marker interface. Or am I missing something here?

Reply:
Good catch, seems I was drunk when writing the docs. I will fix this.

Reply:
Done :-)